File: sample/luastro/examples/hello.lua
print("hello, luastro")
print(1 + 2 * 3)
print(2^10)

local function fib(n)
  if n < 2 then return n end
  return fib(n - 1) + fib(n - 2)
end
print(fib(10))

local t = {10, 20, 30, name = "x"}
print(#t, t[1], t[2], t.name)

for i = 1, 5 do io.write(i, " ") end
print()

File: sample/jstro/examples/hello.js
console.log("hello, jstro!");
console.log(1 + 2 * 3);
console.log("typeof 42:", typeof 42);

function fib(n) {
  if (n < 2) return n;
  return fib(n - 1) + fib(n - 2);
}
console.log("fib(10) =", fib(10));
